### PR TITLE
Quick proof and test of dAPI3 tutorial

### PR DESCRIPTION
**Pull request by kailash-manasarovar**
- Tidied up some of the language.
- A couple of minor clarifications.

### CODE REVIEW

File: docs/dev/tutorials/api3-usd-paymaster-tutorial.md [hunk 1]
```diff
@@ -1,62 +1,74 @@
...
 - Create the `mockUSDC` token contract and send some tokens to a new wallet.
 - Send a `greet` transaction to update the greeting from the newly created wallet via the paymaster. Although the transaction normally requires ETH to pay the gas fee, our paymaster executes the transaction in exchange for the same USDC value.
+- Utilize API3 data feeds within a paymaster.
 
-- Utilize API3 Data Feeds within a paymaster.
+## Prerequisites
 
-## Using API3's self-funded dAPIs with zkSync paymaster example to pay gas fee in USDC on zkSync Era. 
+- Make sure your machine satisfies the [system requirements](https://github.com/matter-labs/era-compiler-solidity/tree/main#system-requirements).
+- You are already familiar with deploying smart contracts on zkSync. If not, please refer to the first section of the [quickstart tutorial](../building-on-zksync/hello-world.md).
```
Comment (uF4No) [inline docs/dev/tutorials/api3-usd-paymaster-tutorial.md:13]:
```suggestion
- You are already familiar with deploying smart contracts on zkSync Era. If not, please refer to the first section of the [quickstart tutorial](../building-on-zksync/hello-world.md).
```

File: docs/dev/tutorials/api3-usd-paymaster-tutorial.md [hunk 2]
```diff
@@ -87,12 +99,15 @@ contract MyERC20 is ERC20 {
...
 
-### Paymaster solidity contract
+### 2. Paymaster solidity contract
 
+Create `MyPaymaster.sol` in the `/contracts` directory. 
 
-2. We can now create our paymaster contract `MyPaymaster.sol` under `/contracts` directory. As it is a custom implementation of the [zkSync paymaster](/docs/dev/tutorials/custom-paymaster-tutorial.html) contract that uses dAPIs, make sure you've followed the guide to understand the basics of paymasters.
+:::tip Tip
+As it is a custom implementation of the zkSync Era paymaster contract that uses dAPIs, make sure you follow the [zkSync paymaster](/docs/dev/tutorials/custom-paymaster-tutorial.html) to understand the basics of paymasters.
+:::
```
Comment (uF4No) [inline docs/dev/tutorials/api3-usd-paymaster-tutorial.md:110]: I'd delete this tip. It points to the other paymaster tutorial but it adds no value in my opinion

Comment (kailash-manasarovar) [reply docs/dev/tutorials/api3-usd-paymaster-tutorial.md:110]: let's do it

File: docs/dev/tutorials/api3-usd-paymaster-tutorial.md [hunk 3]
```diff
@@ -1,62 +1,74 @@
...
+- Create a paymaster that takes `mockUSDC` as gas to cover the transaction cost.
 - Create the `mockUSDC` token contract and send some tokens to a new wallet.
 - Send a `greet` transaction to update the greeting from the newly created wallet via the paymaster. Although the transaction normally requires ETH to pay the gas fee, our paymaster executes the transaction in exchange for the same USDC value.
+- Utilize API3 data feeds within a paymaster.
 
-- Utilize API3 Data Feeds within a paymaster.
+## Prerequisites
 
-## Using API3's self-funded dAPIs with zkSync paymaster example to pay gas fee in USDC on zkSync Era. 
+- Make sure your machine satisfies the [system requirements](https://github.com/matter-labs/era-compiler-solidity/tree/main#system-requirements).
```
[the remark this answers is not in the document]
Comment (uF4No) [reply docs/dev/tutorials/api3-usd-paymaster-tutorial.md:12]: ok, added requirements back specifying these are for zksolc

File: docs/dev/tutorials/api3-usd-paymaster-tutorial.md [hunk 4]
```diff
@@ -87,12 +100,15 @@ contract MyERC20 is ERC20 {
...
 
-### Paymaster solidity contract
+### 2. Paymaster solidity contract
 
+Create `MyPaymaster.sol` in the `/contracts` directory. 
 
-2. We can now create our paymaster contract `MyPaymaster.sol` under `/contracts` directory. As it is a custom implementation of the [zkSync paymaster](/docs/dev/tutorials/custom-paymaster-tutorial.html) contract that uses dAPIs, make sure you've followed the guide to understand the basics of paymasters.
+:::tip Tip
+As it is a custom implementation of the zkSync Era paymaster contract that uses dAPIs, make sure you follow the [zkSync paymaster](/docs/dev/tutorials/custom-paymaster-tutorial.html) to understand the basics of paymasters.
+:::
```
Comment (uF4No) [inline docs/dev/tutorials/api3-usd-paymaster-tutorial.md:111]:
```suggestion
```

**Review by uF4No — APPROVED** [review]

**Pull request merged**